pkg/storage/filesystem/version.go

```diff
@@ -1,4 +1,4 @@
-package filepath
+package filesystem
```

Comment:
what do you think about putting these functions in an internal/filesystem package? i'm a little nervous about making them part of the public api of the library...

Reply:
This has been refactored out